ansible/kubernetes.yml

```diff
@@ -9,12 +9,35 @@
       listen:
         - systemd_read_configs
   post_tasks:
+    - name: ansible_debug_example_1
+        To execute this task, call
```

Comment:
I think my previous comment slipped through :(

I'd remove mentions to vagrant from the description of this task, because Ansible has no notion of Vagrant. Additionally, we say the same thing in the README, so no need to specify it here.